django/example/views/buy_subscription.py

```diff
@@ -23,9 +23,9 @@ class BuySubscriptionView(Injector):
     render = functions.Render.do
 
     @operation
-    def get(show_prices, category_id, render):
+    def get(show_prices, user, category_id, render):
 
-        return render(show_prices(category_id, None))
+        return render(show_prices(user, category_id, None))
```

Comment:
If it just attributes access, we should be able to pass it directly into context. No need to go through the story.

```python
context = show_prices(category_id, None)
context["account_balance"] = ctx.user.profile.balance
return render(context)
```

Reply:
If I got you right, I fixed it.